.github/workflows/ci.yml

```diff
@@ -22,8 +26,39 @@ jobs:
     - name: Install Newman dependency
       run: npm install --prefix tests/postman/
 
-    - name: Run the API and Postman's tests
-      run: |
-        cd tests/postman/ && newman run *
+    - name: Start Netlify Dev for local function testing
+      run:  npm run netlify-dev & sleep 10 && cd tests && node newman.js 
+      env:
+        STRIPE_TEST_KEY: sk_test_7LEesZrYrgDtyxsc6Hy5i0lS00GVFEbmqb
+        ENDPOINT: http://localhost:5050
+
+    - name: Run tests against deployed preview
+      run:  cd tests && node newman.js 
       env:
-        CI: true
+        STRIPE_TEST_KEY: sk_test_7LEesZrYrgDtyxsc6Hy5i0lS00GVFEbmqb
```

Comment:
@vickywane this stripe key should be read from a deployment environment variable. Not included in a public file!
See this: https://docs.github.com/en/actions/reference/environment-variables

Reply:
The key is publicly provided by Stripe for running tests against their API. I would move it into a environment file

Reply:
This has been moved and now supplied with Github Secrets